catalogs/catalog-messaging-kafka/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+description = "catalog-messaging-kafka"
+
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+dependencies {
+  implementation(project(":api"))
+  implementation(project(":core"))
+  implementation(project(":common"))
+}
+
+tasks {
+  val copyDepends by registering(Copy::class) {
+    from(configurations.runtimeClasspath)
+    into("build/libs_all")
+  }
+
+  val copyCatalogLibs by registering(Copy::class) {
+    dependsOn(copyDepends, "build")
+    from("build/libs_all", "build/libs")
```

Comment:
I think you should follow what  @yuqi1129 just changed in #2397

Reply:
rebase and fixed